src/BloomExe/web/ServerBase.cs

```diff
@@ -42,7 +68,7 @@ public abstract class ServerBase : IDisposable
 		/// <summary>
 		/// Pool of threads that pull a request from the _queue and processes it
 		/// </summary>
-		private readonly Thread[] _workers;
+		private readonly List<Thread> _workers = new List<Thread>();
```

Comment:
Anyone think we need more locking on _workers now it can be modified? I think the initial spin-up is single-threaded, and I doubt we'll be spinning up new threads while shutting down. The other spin-up call IS inside a lock. So I think we're OK, but I'm willing to be persuaded otherwise.